VAMobile/src/screens/HealthScreen/Appointments/UpcomingAppointments/UpcomingAppointmentDetails.tsx

```diff
@@ -183,7 +184,10 @@ const UpcomingAppointmentDetails: FC<UpcomingAppointmentDetailsProps> = ({ route
     if (appointmentType === AppointmentTypeConstants.VA_VIDEO_CONNECT_HOME && !isAppointmentCanceled) {
       const onPrepareForVideoVisit = () => {
         dispatch(clearAppointmentCancellation())
-        navigateTo('PrepareForVideoVisit')()
+
+        if (waygateNativeAlert('WG_PrepareForVideoVisit')) {
```

Comment:
Just added a comment on Binny's similar PR about potentially replacing this repeated pattern with a new hook: https://github.com/department-of-veterans-affairs/va-mobile-app/pull/7250/files#r1391705058

Reply:
I like the idea of removing this pattern. I left a comment on that thread